nft/nfts.yaml

```diff
@@ -773,6 +866,67 @@ components:
       schema:
         type: string
       required: false
+    startBlock:
+      name: startBlock
+      description: String - The block number to start fetching NFT sales data from. Allowed values are decimal integers and "latest". Defaults to "latest".
+      in: query
+      schema:
+        type: string
+        default: 'latest'
+      required: false
+    startLogIndex:
+      name: startLogIndex
+      description: Integer - The log index within the startBlock to start fetching NFT sales data from.
+      in: query
+      schema:
+        type: integer
+      required: false
+    startBundleIndex:
+      name: startBundleIndex
+      description: Integer - The index of an NFT within a sale bundle to start fetching NFT sales data from.
```

Comment:
i probably am missing context here but is "sale bundle" a common this or something we decided?

Reply:
This is terminology from Seaport but the interface can also be applied to other marketplaces that support batched/bundled NFT sales.